docs/community/com-cluster/com-cluster-setup.md

```diff
@@ -84,6 +87,12 @@ log-path = "<log-directory>"
 | `advertise-peer-address` | String | Node address and port used by nodes in the cluster to connect. |  | FeatureBase will use `listen-peer-address` values if undefined. |
 | `initial-cluster` | String | A comma-separated list of `advertise-peer-address` values for each node in the cluster, beginning with the parent node. | `featurebase1=http://localhost:10301` | This list must be identical on all nodes in the cluster. |
 
+### `[anti-entropy]` parameters
```

Comment:
Added back in as requested :)

Reply:
Plus built and retested for broken links and it's all tickety-boo
The only odd thing is in com-cluster-setup.md there's a table that's forced to scroll sideways. I don't like that much BUT I don't know how to fix >.<